.github/workflows/pr-test-lint.yml

```diff
@@ -135,13 +135,19 @@ jobs:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
           filterOutClosed: true
           filterOutDraft: true
-      - name: Approve and merge a PR
-        run: |
-          gh pr review ${{ steps.pr.outputs.number }} --approve
-          gh pr merge ${{ steps.pr.outputs.number }} --squash --merge
-        env:
-          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
-
+      # fetch a token for the mondoo-mergebot app
+      - name: Generate token
+        id: generate-token
+        uses: tibdex/github-app-token@v1
```

Comment:
There is now an official action for this:
https://github.com/actions/create-github-app-token

We should switch to this one. It also has a more recent release.

Reply:
switched